sdk/core/System.BinaryData/api/System.BinaryData.netstandard2.0.cs

```diff
@@ -0,0 +1,27 @@
+namespace System
+{
+    public partial class BinaryData
+    {
+        public BinaryData(byte[] data) { }
+        public BinaryData(object jsonSerializable, System.Text.Json.JsonSerializerOptions? options = null, System.Type? type = null) { }
```

Comment:
The _jsonSerializable_ parameter should be nullable. Null objects are always allowed and will convert to the JSON payload `null`. (Same comment on static factory.)

Reply:
Seems as though this is not possible for the generic method in C# 8?

Reply:
Ah, ignore the comment on the generic method. If the caller wants to allow nullable they'd specify it at the call site via `.Serialize<MyObject?>(/* ... */)` or similar.

Reply:
Interesting.. I have a test that passes null but the compiler didn't complain even before changing the param to be nullable.

Reply:
Huh. Maybe your test projects don't enable the nullable feature? I think we disabled nullability checking throughout all of our runtime test projects due to the sheer amount of noise it would generate.